### PR TITLE
feat: create Settings/Accounts/Calendars page

### LINKED ISSUE #4059: Accounts/Calendars - Create page
## Context

For this issue:
- Create an empty page with the route: `/settings/accounts/calendars`.
- Remove the "Soon" pill from the Accounts/Calendars submenu in the Settings menu. Make it navigate to `/settings/accounts/calendars` on click.
- Use the breadcrumb component from #1814.

## Figma

Image

https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty?type=design&node-id=22605%3A135812&mode=dev

### PR DESCRIPTION
**Pull request by thaisguigon**
Closes #4059

### CODE REVIEW

File: packages/twenty-front/src/pages/settings/accounts/SettingsAccountsCalendars.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { SettingsPageContainer } from '@/settings/components/SettingsPageContainer';
```
Comment (charlesBochet) [inline packages/twenty-front/src/pages/settings/accounts/SettingsAccountsCalendars.tsx:1]: @thaisguigon Could you create a story for this page?

File: packages/twenty-front/src/modules/settings/hooks/useSettingsNavItem.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { useMatch, useResolvedPath } from 'react-router-dom';
+
+import { getSettingsPagePath } from '@/settings/utils/getSettingsPagePath';
+import { SettingsPath } from '@/types/SettingsPath';
+
+export const useSettingsNavItem = <Path extends SettingsPath>({
```
Comment (charlesBochet) [inline packages/twenty-front/src/modules/settings/hooks/useSettingsNavItem.ts:6]: I feel the useSettingsNavItem is not clear: can we make it computeSettingsNavItemConfig?
Also could you add an unit test on this one?

**Review by charlesBochet — APPROVED** [review]
Thanks @thaisguigon. I've left two comments :)

File: packages/twenty-front/src/pages/settings/accounts/SettingsAccountsCalendars.tsx (hunk 1 above)
Comment (thaisguigon) [reply packages/twenty-front/src/pages/settings/accounts/SettingsAccountsCalendars.tsx:1]: Done!

File: packages/twenty-front/src/modules/settings/hooks/useSettingsNavItem.ts (hunk 2 above)
Comment (thaisguigon) [reply packages/twenty-front/src/modules/settings/hooks/useSettingsNavItem.ts]: I have replaced `useSettingsNavItem` with a `SettingsNavigationDrawerItem` component. It should be covered by the Settings pages stories. Do we want to add a specific story for the `SettingsNavigationDrawerItems` component as well?

Comment (charlesBochet) [reply packages/twenty-front/src/modules/settings/hooks/useSettingsNavItem.ts]: It does not seems to be necessary at this point. We might introduce a Settings coverage later to make sure that everything is covered. It seems that this is captured by other stories so we are good to go

**Review by charlesBochet — APPROVED** [review]

**Pull request merged**